terraform/azure_fxci/runbooks/nv6_audit_delete.ps1

```diff
@@ -0,0 +1,24 @@
+# Reference: https://mozilla-hub.atlassian.net/browse/RELOPS-231
+# Temp solution to remove old NV6 VMs
+
+$connection = Get-AutomationConnection -Name AzureRunAsConnection
+$connectionResult = Connect-AzAccount -ServicePrincipal -Tenant $connection.TenantID -ApplicationId $connection.ApplicationID -CertificateThumbprint $connection.CertificateThumbprint
+
+$current = (get-date -format g)
+$vms = (get-azvm)
+$current = ((Get-Date).ToUniversalTime())
+
+foreach ($vm in $vms) {
+	if ($VM.HardwareProfile.VMSize -like "Standard_NV6") {
```

Comment:
I think you have to put in a `*` inside of a `-like` statement.

Reply:
This is a full string match, so the `*` is not needed. I tend to use `-like` instead of `-match` because `-like` is not case sensitive.